### PR TITLE
Fix tests and sample_response asset

### PR DESCRIPTION
**Pull request by bernardolk**
Just couldn't test MS2DS integration tests because there is no model deployed in dev yet

### CODE REVIEW

File: tests/ms2deepscore/test_ms2deepscore_integration.py [hunk 1]
```diff
@@ -6,20 +6,23 @@
 def test_match_spectra_from_path_with_2_spectra(
     mgf_path_of_2_spectra, ms2deepscore_client
 ):
+    n_best = 3
     result = ms2deepscore_client.load_and_match_spectra(
```
Comment (cereniyim) [inline tests/ms2deepscore/test_ms2deepscore_integration.py]: when we deploy new version of client this test needs to be updated with the new method name, let's not forget this. Maybe add a TODO?

**Review by cereniyim — APPROVED** [review]
Just have one comment, rest looks good

File: tests/ms2deepscore/test_ms2deepscore_integration.py (hunk 1 above)
Comment (carloszanella) [reply tests/ms2deepscore/test_ms2deepscore_integration.py]: It should be updated now already. Probably test is not passing

File: tests/test_spectra_matching.py [hunk 2]
```diff
@@ -57,7 +57,7 @@ def test_format_results(sample_response):
     assert isinstance(results[0], pd.DataFrame)
     assert results[0].index.name == "matches of spectrum-0"
     assert all(results[0]["score"] > 0)
-    assert all(results[0].columns == ["score", "compound_name", "smiles"])
+    assert "metadata" not in results[0].columns.values
```
Comment (carloszanella) [inline tests/test_spectra_matching.py]: very weird assertion. I would rather assert for the columns that are present

**Review by carloszanella — COMMENTED** [review]
un comento per favore

File: tests/ms2deepscore/test_ms2deepscore_integration.py (hunk 1 above)
Comment (bernardolk) [reply tests/ms2deepscore/test_ms2deepscore_integration.py]: Yeah this one isn't tested yet

**Pull request merged**